app/src/main/java/com/myapps/toualbiamine/basketbusiness/Model/User.java

```diff
@@ -0,0 +1,38 @@
+package com.myapps.toualbiamine.basketbusiness.Model;
+
+
+public class User {
+        private String name;
+        private String password;
+        private String email;
+        private boolean isStaff;
+
+    public User() {
+
+}
+
+    public User(String email, String name, String password, Boolean isStaff) {
```

Comment:
`Boolean isStaff` should be `boolean isStaff` to conform with how we define isStaff.